### PR TITLE
chore: remove arm target from tagged releases

### PR DESCRIPTION
**Pull request by lazpavel**
#### Description of changes
- remove arm target from tagged release
- fixes tagged release job

#### Description of how you validated changes
- created a tagged release [here](https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/11492/workflows/78f170b0-4aae-42fc-b7b6-7cae16aceaf6)

#### Checklist
- [x] PR description included

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by awsluja** [comment]
I think I need some more context to know if this won't break other things, for example, the publish.sh script is setup to publish from tagged-release, as well as tagged-release-without-e2e-tests. There must be some reason for that, but I'm not sure why. Also, if there is a root cause, maybe I can help fix it if it's related to CircleCI or credentials

**Review by evcodes — APPROVED** [review]
small question otherwise, looks good to me.

File: .gitignore [hunk 1]
```diff
@@ -23,6 +23,7 @@ packages/amplify-appsync-simulator/public
 packages/amplify-dynamodb-simulator/emulator
 packages/amplify-dynamodb-simulator/__test__/dynamodb-data
 .vscode/*
+.history/*
```
[the remark this answers is not in the document]
Comment (lazpavel) [reply .gitignore:26]: I use a vs code plugin that keeps local history of changed files

**Comment by lazpavel** [comment]
closing PR as the root cause is having references to different versions of the CDK package causing size increase

**Pull request closed without merging**